### ISSUE TITLE
Notification should update immediately

### ISSUE THREAD

**Issue by timxx** [tualatrix/LinuxNativeNotification#9]
Suppose when you have many notification frequently, 
LinuxNativeNotification is too delay for the newly message

It should update immediately rather than display prev messages one by one

**Comment by tualatrix** [comment]
I agree with you. I should add option to make it update immediately.

**Comment by semente** [comment]
It is not working on my GNOME 3.4. Do I miss some configuration or I just have to install and it must work? Is there a debug log or something? Thanks!
